src/chariot/database/Database.py

```diff
@@ -0,0 +1,60 @@
+import abc
+from typing import List
+
+
+class Database(metaclass=abc.ABCMeta):
```

Comment:
Shouldn't this class be called `DatabaseWriter`? Since we are never reading from the database its name should describe the limitations of its responsibility.